src/overset/oversetUtilities.F90

```diff
@@ -1670,6 +1666,12 @@ subroutine checkOverset(level, sps, totalOrphans, printBadCells)
                                         ! This cell is an orphan:
                                         n = n + 1
                                         orphans(:, n) = (/ii, jj, kk/)
+                                        if (printBadCells) then
```

Comment:
Did you mean to print something here? The logic variables suggest that there is going to be something printed.

Reply:
Ok, I see yeah, I think breaking this up into smaller functions with clear responsibilities, i.e., as separate call might make sense, but we can keep for now.